File: loop.rb
puts "1. UNCONDITIONAL / UNCONDITIONAL"
for i in 1..5 do
 puts "i = " + i.to_s
 break if i > 2
end

puts "\n2. LABELED / UNLABELED"
for i in 1..5 do
    puts "\ni = " + i.to_s + "\n"
    for j in 1..5 do
        print "j = " + j.to_s + ", "
        break if j > 2
    end
end

File: loop.dart
void main()
{
  print("1. UNCONDITIONAL / UNCONDITIONAL\n");
  var a = 1; 
  while( a < 5 ) 
  { 
    print("a = $a");
    if ( a > 2 ) 
    { 
       break ;    
    } 
    a++; 
  }
  
  print("\n2. LABELED / UNLABELED");
  var i = 1; 
  while( i < 5 ) 
  { 
    print("\ni = $i");
    var j = 1; 
    while( j < 5 ) 
    { 
      print("j = $j");
      if ( j > 2 ) 
      { 
         break ;    
      } 
      j++; 
    }
    i++;
  }
}